### ISSUE TITLE
Plugin Directory Checklist

### ISSUE THREAD

**Issue by oguzkocer** [wordpress-mobile/WordPress-Android#7206]
I am creating a checklist for myself and @nbradbury to remind us what's missing or not working properly right now:

- [x] Check that the fluxc event we get in plugin detail page (and maybe in other activity/fragments) the one we are interested in. For example, `onWPOrgPluginFetched` is handled regardless of which plugin we are showing
- [x] Add back pull to refresh to plugin detail page (check whether that might break [this bit](https://github.com/wordpress-mobile/WordPress-Android/pull/7163#discussion_r166653248))
- [x] Update tap target for ***Manage*** and ***See All*** buttons to at least 48dp to match [Material guidelines](https://material.io/guidelines/layout/metrics-keylines.html#metrics-keylines-touch-target-size).  See screenshot below for illustration.
![plugin_button](https://user-images.githubusercontent.com/3827611/36292562-c945aa06-128f-11e8-8327-5aa1eb403661.png)
- [x] Add shadow to toolbar like Plugin Directory view like Plugin Detail view.  See screenshots below for illustration.
![plugin_shadow](https://user-images.githubusercontent.com/3827611/36292478-53a52326-128f-11e8-915c-e587fd4181f4.png)

**Comment by oguzkocer** [comment]
@theck13 Do you think you can tackle those 2 minor issue you've added?

I am not exactly sure how we can have 48dp min width & height for those buttons while keeping the design the same. If the design is supposed to change as well, we should check in with a designer about that.

And for the shadow, I am not even sure what I am supposed to see in those screenshots :(

**Comment by theck13** [comment]
> I am not exactly sure how we can have 48dp min width & height for those buttons while keeping the design the same.

I updated the view layout in https://github.com/wordpress-mobile/WordPress-Android/pull/7254 to make the button tap target 48dp.  The overall design remains the same.  We can pull in a designer if you would like too.

> And for the shadow, I am not even sure what I am supposed to see in those screenshots :(

It's very subtle, but there is a shadow below the light blue part of the toolbar of the Detail screenshot to signify it is higher on the z-axis than the content below it.  I added the shadow in https://github.com/wordpress-mobile/WordPress-Android/pull/7253 by removing the `setElevation(0)` call.
